### PR TITLE
docs(em): fix netplan ipv6 config

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx [hunk 1]
```diff
@@ -43,20 +43,19 @@ Since the release of version 18.04 (Bionic Beaver), Ubuntu has used Netplan for
...
-            dhcp6: no
-            addresses: [fail.over.ip.address/32]
-            gateway4: 62.210.0.1
+    ethernets:
+        ens18:
+            addresses:
+            - [ "fail.over.ip.address/32" ]
             nameservers:
-              addresses: [ "51.159.47.28", "51.159.47.26" ] # Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performances (https://www.scaleway.com/en/docs/console/account/reference-content/scaleway-network-information/#dns-cache-servers/)
+                addresses: [ "51.159.47.28", "51.159.47.26" ] # Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performances (https://www.scaleway.com/en/docs/console/account/reference-content/scaleway-network-information/#dns-cache-servers/)
```
Comment (bene2k1) [inline dedibox-network/ip-failover/how-to/configure-network-virtual-machine.mdx:51]:
```suggestion
              addresses: [ "51.159.47.28", "51.159.47.26" ] # Replace the IP of the DNS cache server with the one located in the same physical location as your machine for optimal performances (https://www.scaleway.com/en/docs/console/account/reference-content/scaleway-network-information/#dns-cache-servers/)
```

File: bare-metal/elastic-metal/how-to/configure-flexible-ipv6.mdx [hunk 2]
```diff
@@ -87,7 +87,10 @@ It replaces the classical network configuration with new configuration files, wr
     ```
     sudo netplan apply
     ```
-
+4. Test the connection with a ping to an external host
```
Comment (nerda-codes) [inline bare-metal/elastic-metal/how-to/configure-flexible-ipv6.mdx:90]:
```suggestion
4. Test the connection with a ping to an external host:
```

File: bare-metal/elastic-metal/how-to/configure-ipv6-hypervisor.mdx [hunk 3]
```diff
@@ -1,102 +1,137 @@
...
+   ip a
+   ```
+   Take note of your network interface name (e.g., `ens18`).
 
-1. Log into your virtual machine and open the network configuration file `/etc/netplan/01-netcfg.yaml` in a text editor of your choice, for example `nano`:
+3. Find your IPv6 gateway:
+   ```sh
+   ip -6 route
+   ```
+   Look for the line that specifies the default route, which will resemble:
```
Comment (nerda-codes) [inline bare-metal/elastic-metal/how-to/configure-ipv6-hypervisor.mdx:46]:
```suggestion
   Look for the line that specifies the default route. It should look like the following:
```

File: bare-metal/elastic-metal/how-to/configure-ipv6-hypervisor.mdx [hunk 4]
```diff
@@ -1,102 +1,137 @@
...
+      gateway LINK_LOCAL_IPv6_GATEWAY
+    ```
+    Replace `FLEXIBLE_IPv4`, `FLEXIBLE_IPv6`, and `LINK_LOCAL_IPv6_GATEWAY` with your actual IP addresses.
+
+3. Set the DNS resolver:
+    ```sh
+    sudo nano /etc/resolv.conf
     ```
-5. Set the DNS resolver by adding the following lines to the file. Then save it and exit the text editor:
+    Add the following lines:
```
Comment (nerda-codes) [inline bare-metal/elastic-metal/how-to/configure-ipv6-hypervisor.mdx:116]:
```suggestion
 4. Add the following lines, then save and exit the file.
```

File: bare-metal/elastic-metal/how-to/configure-ipv6-hypervisor.mdx [hunk 5]
```diff
@@ -1,102 +1,137 @@
...
+    nameserver 51.159.47.26
     ```
 
-    <Message typ="tip">
-      These DNS resolvers are operated by Scaleway. You can also set your preferred alternative DNS resolvers in the file.
-    </Message>
-6. Activate the network on your virtual machine by running the following code:
-    ```
-    ifup ens18
+4. Activate the network configuration:
```
Comment (nerda-codes) [inline bare-metal/elastic-metal/how-to/configure-ipv6-hypervisor.mdx:122]:
```suggestion
5. Activate the network configuration:
```

File: bare-metal/elastic-metal/how-to/configure-ipv6-hypervisor.mdx [hunk 6]
```diff
@@ -105,17 +140,60 @@ Since the release of version 18.04 (Bionic Beaver) Ubuntu has switched to Netpla
...
-    ```
-
+    Replace `FLEXIBLE_IPv4`, `FLEXIBLE_IPv6`, and `virtual:mac:address` with your actual IP addresses and MAC address.
 
+3. Bring up the network interface:
+    ```sh
+    sudo ifup ens18
+    ```
+
+4. Find the IPv6 Gateway:
```
Comment (nerda-codes) [inline bare-metal/elastic-metal/how-to/configure-ipv6-hypervisor.mdx:159]:
```suggestion
4. Find the IPv6 gateway:
```

File: bare-metal/elastic-metal/how-to/configure-ipv6-hypervisor.mdx [hunk 7]
```diff
@@ -105,17 +140,60 @@ Since the release of version 18.04 (Bionic Beaver) Ubuntu has switched to Netpla
...
+4. Review your network logs:
+   ```sh
+   sudo journalctl -u systemd-networkd
+   ```
+
+5. Restart network services:
+   ```sh
+   sudo systemctl restart systemd-networkd
+   ```
+6. Verify Link-Local Address:
```
Comment (nerda-codes) [inline bare-metal/elastic-metal/how-to/configure-ipv6-hypervisor.mdx:195]:
```suggestion
6. Verify the link-local address:
```

**Review by nerda-codes — COMMENTED** [review]
BONJOUR BENERDIKT

**Review by nerda-codes — APPROVED** [review]

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**